examples/values-example.yaml

```diff
@@ -23,13 +23,20 @@ clusterGroup:
       annotations:
         openshift.io/cluster-monitoring: "true"
         owner: "namespace owner"
-  - application-ci
+  - application-ci:
+      operatorgroup:
+        targetNamespaces:
+          - application-ci
+          - other-namespace
```

Comment:
I would suggest decoupling `operatorgroup` and `targetNamespaces`
Also, be consistent with camelCase.

```suggestion
      operatorGroup: true # true being the default anyway
      targetNamespaces:
        - application-ci
        - other-namespace
```

Reply:
Thanks for the suggestion and it totally makes sense.  Updated code to support this.